### PR TITLE
Add styles for page recover-password

### PR DESCRIPTION
**Pull request by AlejandroCuriel**
# Description
 - What?: Add styles to the view generated by the devise gem for password recovery
 - Why?: So that the user can better interact with the form and be more attractive to him. See #64 for more information.

  
  ## Screenshots 
Screenshot from 2022-05-19 00-22-54

### CODE REVIEW

File: app/assets/stylesheets/components/_buttons.scss [hunk 1]
```diff
@@ -7,3 +7,18 @@
...
+.btn-reset-password {
+  font-weight: 700;
+  font-size: large;
+  border: none;
+  padding: 1rem 2rem;
+  margin: 2rem 0;
+  background-color: $profile-button;
+  border-radius: 4rem;
+  color: $white;
+
```
Comment (germanruzca) [inline app/assets/stylesheets/components/_buttons.scss:20]: Sort this

File: app/assets/stylesheets/pages/_new_password.scss [hunk 2]
```diff
@@ -0,0 +1,25 @@
...
+}
+
+.reset__password-box {
+  max-width: 600px;
+  padding: 0 5rem 5rem 5rem;
+  margin: 0 auto;
+  border: 2px solid rgb(0, 0, 0);
+  border-radius: 5rem;
+  background-color: $white ;
+
```
Comment (germanruzca) [inline app/assets/stylesheets/pages/_new_password.scss:16]: Sort this

**Review by germanruzca — CHANGES_REQUESTED** [review]
Looks good, but I think the input should have a style like others forms.

File: app/assets/stylesheets/components/_buttons.scss (hunk 1 above)
Comment (JuanequeX) [reply app/assets/stylesheets/components/_buttons.scss:20]: i aggre with @germanruzca

File: app/assets/stylesheets/pages/_new_password.scss (hunk 2 above)
Comment (JuanequeX) [reply app/assets/stylesheets/pages/_new_password.scss:16]: I aggre with @germanruzca

File: app/views/devise/passwords/new.html.erb [hunk 3]
```diff
@@ -1,16 +1,21 @@
...
-<%= form_for(resource, as: resource_name, url: password_path(resource_name), html: { method: :post }) do |f| %>
-  <%= render "devise/shared/error_messages", resource: resource %>
+    <%= form_for(resource, as: resource_name, url: password_path(resource_name), html: { method: :post }) do |f| %>
+      <%= render "devise/shared/error_messages", resource: resource %>
 
-  <div class="field">
-    <%= f.label :email %><br />
-    <%= f.email_field :email, autofocus: true, autocomplete: "email" %>
-  </div>
+      <div class="field">
```
Comment (JuanequeX) [inline app/views/devise/passwords/new.html.erb:9]: This input would be have a better classname to be more specific what it is. What do tou think?

**Review by JuanequeX — CHANGES_REQUESTED** [review]

**Comment by JuanequeX** [comment]
don't forget to solve your conflict merge through command lines using `rebase --continue`

**Review by YilKanda — COMMENTED** [review]
Check the observations that have been made to you.

**Review by germanruzca — APPROVED** [review]

**Review by nallely64 — APPROVED** [review]

**Review by YilKanda — APPROVED** [review]
LGTM!

**Review by germanruzca — APPROVED** [review]

**Review by Javi1499 — APPROVED** [review]

**Pull request merged**